packages/backend/src/database/entities/openIdIdentities.ts

```diff
@@ -8,6 +8,8 @@ export type DbOpenIdIdentity = {
     user_id: number;
     created_at: Date;
     email: string;
+    refresh_token?: string;
+    scope?: string;
```

Comment:
Was wondering whether to store the raw or parsed version. So should this be a list of `scopes` ? The benefit of storing raw is we don't need to bother parsing it upfront and can deal with it later. The downside is that we might have some weird `scope` strings in the database

I don't mind either way